paket.dependencies

```diff
@@ -33,6 +35,8 @@ group Build
 
 group Test
 
+    framework: netcore2.0
```

Comment:
We need `netcore2.0` here to satisfy the dependencies like xUnit.net and Unquote, is that right? 😃

Reply:
What happens if we make this `netstandard2.0`, why not even `1.6`? (Just thinking out loud.)

Reply:
The framework restriction is corresponding to frameworks of projects in a solution. If the line is changed to netstandard2.0 or 1.6, all the nuget packages in the Test group cannot be referenced so compile errors will be occurred.

That line is like saying that only  projects under .net core 2.0 are supported. If we need other frameworks side by side, we can declare simply like,

```
framework: netcore2.0, netstandard2.0, netstandard1.6
```

Reply:
So perhaps `netcore1.6` is what we need? (Instead of `netcore2.0`.)

Reply:
This is correct for the test projects, they should target a concrete framework, not standard 👍

Reply:
@Porges, sounds good. I'm now looking again at https://github.com/Microsoft/visualfsharp/issues/3680#issuecomment-363698804 😃